source/usage-examples/changeStream.txt

```diff
@@ -9,59 +9,59 @@ Open a Change Stream
 
 The ``watch()`` method  optionally takes a :manual:`pipeline
 </reference/operator/aggregation-pipeline/>`, an array of
 :manual:`stages </changeStreams/#modify-change-stream-output>`, as the
-first parameter to filter the :manual:`change events
-</reference/change-events/>` output. Also, ``watch()`` can take an
-additional options object as the second parameter. Set the
-``fullDocument`` field of the additional options object to the string 
-``"updateLookup"`` to receive an event that contains the changes to the
-document as well as the entire altered document. Alternatively, you can
-set ``fullDocument`` to the string ``"default"`` to only receive the
-altered document, but not the changes to the document. 
+first parameter to filter and transform the :manual:`change events
+</reference/change-events/>` output. ``watch()`` accepts an additional
+options object as the second parameter. By default, change streams only
+return the fields modified by an update operation, instead of the entire
+updated document. You can configure your Change Stream to also
+return the most current version of a document by setting the
+``fullDocument`` field of the options object to ``"updateLookup"``.
 
 Process the Change Stream Events
 --------------------------------
 
-You can capture events from a ``Change Stream`` instance by using a listener
-function. Call the ``watch()`` command to get a ``Change Stream`` instance. Add
-your listener function by calling the `EventEmitter.on()
+You can capture events from a Change Stream with a listener
+function. Call the ``watch()`` command to get a ``ChangeStream`` instance.
+Add your listener function by calling the `EventEmitter.on()
 <https://nodejs.org/api/events.html#events_emitter_on_eventname_listener>`_
-method on that instance. Set the value of the first parameter to the
-string ``'change'`` and add your :mdn:`callback function
-<Glossary/Callback_function>` as the second parameter.  The callback
-triggers when a ``change event`` is emitted, providing the next available
-document. You can specify logic in the callback to process the event
-document when it is received. 
+method on that instance. Pass the string ``change`` as the first parameter
+and add your :mdn:`callback function <Glossary/Callback_function>` as
+the second parameter.  The callback triggers when a change event is
+emitted, providing the next available document. You can specify logic in
+the callback to process the event document when it is received.
 
 Call the :node-api:`close() </ChangeStream.html#close>` method on the
-``Change Stream`` instance to stop processing ``change events``. This method
-closes the ``Change Stream`` and frees resources.
+``ChangeStream`` instance to stop processing change events. This method
+closes the Change Stream and frees resources.
 
 Example
 -------
 
-The following example opens a ``Change Stream`` on the ``movies``
+The following example opens a Change Stream on the ``movies``
```

Comment:
I think if we want to use ``ChangeStream`` for the instance and Change Stream to describe the term, it should not be a proper noun in the latter, e.g. "change stream" which has an analog: "change event"